include/swift/AST/Decl.h

```diff
@@ -5158,6 +5161,13 @@ class VarDecl : public AbstractStorageDecl {
   }
 };
 
+enum class ParamSpecifier : uint8_t {
```

Comment:
'long as we're here, can we pick a better name than "Specifier"?

Reply:
Sure, I'm all ears :) I believe one of your refactorings introduced the name Specifier, but I might be wrong :)

Reply:
I did it back then because the it's the name the Swift Book's "grammar" gives it.